### ISSUE TITLE
Accessibility: Low Contrast on CTA Buttons (Likely Requires Design Review)

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1046]
## Issue
All of the "Start Earning" buttons on the landing page (outside of the video player) have label text with an insufficient contrast against the background.

## Expected Behavior
- [WCAG 2.1 AA 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG22/#contrast-minimum) requires a contrast ratio of **at least 4.5:1** for regular sized text on buttons and 3:1 contrast is used for large text (greater than 14pt **and** bold or 18+pt).
- Update the button's background color to increase luminance contrast. See current ratio https://www.myndex.com/APCA/?BG=000000&TXT=7a91db&DEV=G4g&BUF=A22 **and/or**
- Update to 19px and font weight to at least 700 (bold)
## Acceptance Criteria
- [ ] Verify new color meets WCAG accessibility guidelines at minimum and optionally APCA as best practice.
- [ ] Test with [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/). [APCA Contrast Calculator](https://www.myndex.com/APCA/), and/or [Bridge PCA Contrast Checker](https://www.myndex.com/BPCA/)
- [ ] Ensure sufficient contrast and readability in both light and dark buttons.

_Linked to tracking issue: #456_
